reason: drop localversion in x.y.z_LOCALVERSION, else later numeric comparisons
on `micro_version` fail.
upstream: no

--- open-vm-tools/configure.ac
+++ open-vm-tools/configure.ac
@@ -100,7 +100,7 @@
 	    [AS_HELP_STRING([--with-kernel-release],
 		[specifies the kernel release you want to build against])],
 	    [KERNEL_RELEASE="$withval"],
-	    [KERNEL_RELEASE=`uname -r`])
+	    [KERNEL_RELEASE=`uname -r | sed -e 's/_.*//'`])
 
 AC_ARG_WITH([linuxdir],
 	    [AS_HELP_STRING([--with-linuxdir],
